mmv1/third_party/terraform/services/compute/resource_compute_router_peer_test.go.erb

```diff
@@ -0,0 +1,194 @@
+<% autogen_exception -%>
+package compute_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+
+	<% if version == "ga" -%>
```

Comment:
This file doesn't actually use the compute go library. When it was auto-generated the generator ran goimports on it removing unnecessary imports.

To address this I would remove L16-20 here altogether.

Reply:
Removed and pushed the change